components/src/core/list-item-tag/list-item-tag.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { TextProps, TextStyle, StyleProp, StyleSheet } from 'react-native';
+import { Theme, useTheme } from 'react-native-paper';
+import Color from 'color';
+import { white } from '@pxblue/colors';
+import { Overline } from '..';
+import { $DeepPartial } from '@callstack/react-theme-provider';
+
+export type ListItemTagProps = TextProps & {
+    /**
+     * Color of the label background. Default is theme.colors.primary
+     **/
+    backgroundColor?: string;
+
+    /**
+     * Color of the label. Default is theme.colors.onBackground for light background,
+     * or white[50] on dark background
+     */
+    fontColor?: string;
+
+    /** The string label of the tag. */
+    label: string;
+
+    /** Style Overrides */
+    styles?: {
+        root?: StyleProp<TextStyle>;
+    };
+
+    /**
+     * Overrides for theme
+     */
+    theme?: $DeepPartial<Theme>;
+};
+
+const listItemTagStyles = (
+    props: ListItemTagProps,
+    theme: Theme
+): StyleSheet.NamedStyles<{
+    root: TextStyle;
+}> =>
+    StyleSheet.create({
+        root: {
+            backgroundColor: props.backgroundColor || theme.colors.primary,
+            color:
+                props.fontColor ||
+                (Color(props.backgroundColor || theme.colors.primary).isLight()
+                    ? theme.colors.onBackground
+                    : white[50]),
+            padding: 0,
+            paddingLeft: 4,
+            paddingRight: 4,
+            borderRadius: 2,
+            fontWeight: 'bold',
```

Comment:
Bold is actually one of the font-weights that should work like this despite the Android font-weight bug in RN right now. For consistency with how we are changing the other weights, we may want to pull these from the theme definitions of the font instead:
```
fontFamily: theme.fonts.bold.fontFamily,
fontWeight: theme.fonts.bold.fontWeight,
```

Reply:
We don't have a "bold" font configured, using medium fonts here instead.

Here's the type definition in RNP:
```ts
export declare type Fonts = {
    regular: Font;
    medium: Font;
    light: Font;
    thin: Font;
};
```

Reply:
Haha bold will work on Android..........but we don't support bold 😢 
Medium should be fine...I'm seeing if we can get the RNP folks to support a bold weight.

Reply:
We technically could add bold to our theme if we want to, but we would have to put in conditional logic wherever we use it in our component library since the default theme from RNP won't have it. And typescript will pitch a fit.